source/board/cy8ckit.c

```diff
@@ -0,0 +1,430 @@
+/*******************************************************************************
+* @file    cy8ckit.c
+* @brief   Board ID definitions for all Cypress povided kits.
+*
+********************************************************************************
+* Copyright (2019) Cypress Semiconductor Corporation
+* or a subsidiary of Cypress Semiconductor Corporation.
+*
+* Licensed under the Apache License, Version 2.0 (the "License"); you may
+* not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+* WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*****************************************************************************/
+
+#include "target_family.h"
+#include "target_board.h"
+
+#include "CyLib.h"
+#include "Pin_VoltageEn.h"
+#include "target_reset.h"
+#include "EEPROM_ModeStorage.h"
+#include "PSOC6xxx.h"
+
+// Maximum count of supported  HW ids
+#define MAX_SUPPORTED_HWID          (13u)
+
+// Silicon ids (To add new clear revion bits)
+#define CY8CPROTO_062_4343W_SIID    (0xE4020002u)
+#define CY8CKIT_062_BLE_SIID        (0xE2070000u)
+#define CY8CKIT_062_WIFI_BT_SIID    (0xE2060000u)
+#define	CYW943012P6EVB_01_SIID      (0xE2060000u)
+#define CY8CKIT_062_4343W_SIID      (0xE4020002u)
+#define CY8CPROTO_062_43012_SIID    (0xE4020102u) // Added revision number to avoid twins
+#define CY8CPROTO_063_BLE_SIID      (0xE2F00000u)
+#define CY8CPROTO_062S2_43012_SIID  (0xE4020002u)
+#define CY8CPROTO_064_SB_SIID       (0xE2620000u) // CYB06447BZI-D54 
+
+// Unique ID record header
+#define UID_PSOC5_HEADER            (0x6970122Eu)
+
+#define P5LP_EEPROM_ROW_SIZE        (16u)
+#define CRC8_2S_COMP_BASE           (0x0100u)
+
+ // Unique ID  record sctruct t
+typedef struct __attribute__((packed)) unique_id_struct
+{
+	// Unique id header (PSoC5 silicon ID) - 4 bytes
+	uint32_t signature;
```

Comment:
Indention looks funky in this file. Please use 4 space indention only, no tabs, for all contributed files.

Reply:
Indentation has been fixed